products/models.py

```diff
@@ -0,0 +1,100 @@
+from django.db import models
+
+from core.models     import Timestamp
+from users.models    import User
+
+# Create your models here.
+class Menu(Timestamp):
+    menu_name   = models.CharField(max_length=100)
+    menu_image  = models.CharField(max_length=100)
+    
+    class Meta:
+        db_table = 'menus'
```

Comment:
다 너무 verbose합니다. 간결하게 만들어주세요!

```suggestion
class Menu(Timestamp):
    name   = models.CharField(max_length=100)
    image_url  = models.CharField(max_length=100)
    
    class Meta:
        db_table = 'menus'
```